source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.ApplyDBMigrationsApp/Scripts/Model/202109071000 Refactor ChargeLink Model.sql

```diff
@@ -0,0 +1,56 @@
+DROP TABLE [Charges].[ChargeLinkPeriodDetails]
+DROP TABLE [Charges].[ChargeLinkOperation]
+DROP TABLE [Charges].[ChargeLink]
+
+CREATE TABLE [Charges].[ChargeLink]
+(
+    Id UNIQUEIDENTIFIER NOT NULL,
+    ChargeId int NOT NULL FOREIGN KEY REFERENCES [Charges].Charge(RowId),
+    MeteringPointId int NOT NULL FOREIGN KEY REFERENCES [Charges].MeteringPoint(RowId)
+        CONSTRAINT [PK_ChargeLink] PRIMARY KEY NONCLUSTERED
+            (
+             ID ASC
+                ) WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+) ON [PRIMARY]
+GO
+
+CREATE INDEX IX_MeteringPointId ON [Charges].[ChargeLink] (MeteringPointId DESC, ChargeId DESC);
+GO
+
+CREATE TABLE [Charges].[ChargeLinkOperation]
+(
+    Id UNIQUEIDENTIFIER NOT NULL,
+    ChargeLinkId UNIQUEIDENTIFIER NOT NULL FOREIGN KEY REFERENCES [Charges].ChargeLink(Id),
+    CorrelationId nvarchar(36) NOT NULL,
+    CustomerProvidedId nvarchar(100) NOT NULL,
+    WriteDateTime DateTime2 NOT NULL DEFAULT(GETDATE())
+        CONSTRAINT [PK_ChargeLinkOperation] PRIMARY KEY NONCLUSTERED
+            (
+             ID ASC
+                ) WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+) ON [PRIMARY]
+GO
+
+CREATE INDEX IX_ChargeLinkId ON [Charges].[ChargeLinkOperation] (ChargeLinkId DESC, WriteDateTime DESC);
+GO
+
+CREATE TABLE [Charges].[ChargeLinkPeriodDetails]
+(
+    Id UNIQUEIDENTIFIER NOT NULL,
+    ChargeLinkId UNIQUEIDENTIFIER NOT NULL FOREIGN KEY REFERENCES [Charges].ChargeLink(Id),
+    StartDateTime DateTime2 NOT NULL,
+    EndDateTime DateTime2,
```

Comment:
As we became aware of today, let's make this: 
```suggestion
    EndDateTime DateTime2 NOT NULL,
```